### DISCUSSION TITLE
Using the Log Viewer with Lumen

### DISCUSSION THREAD

**Discussion by evandrojlle** [opcodesio/log-viewer#237] [Q&A] [answered]
I'm developing an API and after it's published in production, I don't have access to the server anymore, so I want to follow the logs through LogViewer, but my API, as required by the company, is written with Lumen, but I'm having difficulty configuring it LogViewer.

First, when running the php artisan log-viewer:publish command, I got the error:
ERROR  There are no commands defined in the "log-viewer" namespace.
  
![image](https://github.com/opcodesio/log-viewer/assets/652806/9389f3b1-72e5-4ba6-b1ef-d6f83c3f6179)

So I registered the service provider in bootstrap\app.php. After that the error changed to:
In api.php line 6:
Call to undefined method Laravel\Lumen\Routing\Router::name()
![image](https://github.com/opcodesio/log-viewer/assets/652806/0f765ae2-5f65-46b7-b69a-51d8e579a13f)

My question is: How to use LogViewer in my Lumen application?

I am anxiously awaiting help.
Thanks

**Comment by arukompas** [comment, marked as answer]
Hey @evandrojlle , sorry for such a late response, the notification must've gotten lost somewhere :) 

Unfortunately, Log Viewer currently supports Laravel only. It depends on certain features that exist only in the full Laravel framework (not Lumen), which is why you're experiencing these errors.

I can recommend an alternative package - https://github.com/rap2hpoutre/laravel-log-viewer

Hope this helps!
